docs/local-development/debug-config.md

```diff
@@ -0,0 +1,101 @@
+# Debug
+
+## Difficulties
+It is difficult to debug/step-through front-end code so the recommendation is to use browser development tools separately. When using the IDE for debugging and setting breakpoints it should only be for back-end/server code.
+
+## Gotchas with IDE debugging
+The module aliasing (e.g. import * from "shared/...") is what throws off most debuggers and build tools. Usually `tsconfig` files are used by debuggers and you may need to create a separate tsconfig for your local environment so that the IDE knows how to run your app.
+
+## Alternatives
+Another solution is to configure grunt to emit source maps alongside JS and just debug the JS.
+
+## Front-end
+Debugging the front-end might require evaluating MSG's in the state object.
```

Comment:
@bdolor Do you want me to add some additional docs here in your branch directly?

Reply:
yes, please...by all means...feel free